spec/acceptance/wsus_client_spec.rb

```diff
@@ -69,55 +89,75 @@ def create_apply_manifest(params)
     end
   end
 
-  context "server_url =>", {:testrail => ['70185', '70184']} do
+  context "server_url =>", {:testrail => ['70183', '70185', '70184']} do
```

Comment:
I don't actually as we can further check that the we respect the protocol being based on either and don't implicitly change it.  The default that Microsoft recommends is HTTP anyways